### ISSUE TITLE
CONCEPT_RELATIONSHIP INVALID_REASON

### ISSUE THREAD

**Issue by ericaVoss** [OHDSI/Vocabulary-v5.0#82]
**v5.0 20-JUN-16**

I've found 110 relationships where the VALID_END_DATE is not in the future and INVALID_REASON IS NULL.

``` SQL
SELECT *
FROM CONCEPT_RELATIONSHIP cr
WHERE VALID_END_DATE NOT BETWEEN GETDATE() AND '12/31/2099'
AND INVALID_REASON IS NULL
```

Example records

``` text
concept_id_1    concept_id_2    relationship_id valid_start_date    valid_end_date  invalid_reason
40664775    43533307    Concept replaced by 2011-10-01  2012-12-31  NULL
45890797    45890825    Concept replaces    2014-07-01  2014-12-31  NULL
45890802    45890827    Concept replaces    2014-07-01  2014-12-31  NULL
43533245    43533212    Concept replaced by 2012-04-01  2012-06-30  NULL
```

**Comment by ericaVoss** [comment]
This is fixed in 20160817

**Comment by ericaVoss** [comment]
This is fixed in 20161107
